GettingStarted.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
```

Comment:
I would leave out the prezi.com attribution. impress.js is so much more at this point.